.github/CONTRIBUTING.md

```diff
@@ -1,107 +1,84 @@
+
+2. Add the main LPVS repository as an additional Git remote called "upstream":
+   ```sh
+   git remote add upstream https://github.com/samsung/lpvs
+   ```
+
+3. Make sure there is an issue created for the task you are working on. All new features and bug fixes should have an associated issue to provide a single point of reference for discussion and documentation. If the issue already exists, leave a comment on that issue indicating that you intend to work on it. If it doesn't exist, open a new issue for your task. 
+
+    > For small changes or documentation issues, creating an issue is not necessary, and a pull request is sufficient.
+
+4. Fetch the latest code from the main LPVS branch:
+   ```sh
+   git fetch upstream
+   ```
+    You should start at this point for every new contribution to make sure you are working on the latest code.
+
+5. Create a new branch for your feature based on the current LPVS main branch:
+
+    Each separate bug fix or feature addition should have its own branch. Branch names should be descriptive and start with the number of the corresponding issue, if applicable. If you're not fixing a specific issue, you can skip the number. 
+	```sh
+	git checkout upstream/<NAMED_RELEASE>
+	git checkout -b 999-name-of-your-branch-goes-here
+	```
+    Above, <NAMED_RELEASE> can be '1.0.0', etc. - see the list of releases or `main` branch.
+
+6. Write your code and make the necessary changes.
+   - Follow the coding conventions and style guidelines used in the LPVS project.
+   - Write clear, concise, and well-documented code.
+   - Include unit tests to ensure the correctness of your code.
+   - If you're adding a new feature, consider updating the relevant documentation and examples.
+   - If you're fixing a bug, provide a clear explanation of the issue and how your code resolves it.
+
+    Feel free to reach out if you have any further questions or need additional assistance!
+
+7. Update the Release Notes
+    After committing your changes, consider updating the Release Notes file to include a summary of your contribution. If your change fixes a bug, use the following format: Bug #issue_number: a description of the bug fix (Your Name). For enhancements, use Enh #issue_number: a description of the enhancement (Your Name).
+
+    Keep in mind that for minor changes, such as typos or documentation updates, updating the Release Notes may not be necessary.
+
+8. Commit your changes with a descriptive commit message. Make sure to mention the issue number with `#XXX` so that GitHub will automatically link your commit with the issue:
+   ```sh
+   git add path/to/my/file
+   git commit -m "A brief description of this change which fixes #42" --signoff
+   ```
+
+9. Pull the latest LPVS code from upstream into your branch:
+   ```sh
+   git pull upstream main
+   ```
+
+10. Push your code to your forked repository:
+   ```sh
```

Comment:
I don't think the indentation is applied. Can you check?

Reply:
Agree with your opinion.